### ISSUE TITLE
Bug in match-mondo-sources-all-lexical.py

### ISSUE THREAD

**Issue by matentzn** [monarch-initiative/mondo-ingest#390]
```
Traceback (most recent call last):
  File "/work/src/ontology/../scripts/match-mondo-sources-all-lexical.py", line 174, in <module>
    main()
  File "/usr/local/lib/python3.10/dist-packages/click/core.py", line 1157, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.10/dist-packages/click/core.py", line 1078, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.10/dist-packages/click/core.py", line 1688, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.10/dist-packages/click/core.py", line 1434, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.10/dist-packages/click/core.py", line 783, in invoke
    return __callback(*args, **kwargs)
  File "/work/src/ontology/../scripts/match-mondo-sources-all-lexical.py", line 121, in run
    msdf = lexical_index_to_sssom(oi, lexical_index, ruleset=ruleset, meta=meta)
  File "/usr/local/lib/python3.10/dist-packages/oaklib/utilities/lexical/lexical_indexer.py", line 316, in lexical_index_to_sssom
    Converter.from_prefix_map((meta or {}).pop(CURIE_MAP, {})),
AttributeError: 'tuple' object has no attribute 'pop'
make[1]: *** [mondo-ingest.Makefile:432: ../mappings/mondo-sources-all-lexical.sssom.tsv] Error 1
```

**Comment by matentzn** [comment]
After updating all dependencies the issue persists:

```
Traceback (most recent call last):
  File "/work/src/ontology/../scripts/match-mondo-sources-all-lexical.py", line 174, in <module>
    main()
  File "/usr/local/lib/python3.10/dist-packages/click/core.py", line 1157, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.10/dist-packages/click/core.py", line 1078, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.10/dist-packages/click/core.py", line 1688, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.10/dist-packages/click/core.py", line 1434, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.10/dist-packages/click/core.py", line 783, in invoke
    return __callback(*args, **kwargs)
  File "/work/src/ontology/../scripts/match-mondo-sources-all-lexical.py", line 121, in run
    msdf = lexical_index_to_sssom(oi, lexical_index, ruleset=ruleset, meta=meta)
  File "/usr/local/lib/python3.10/dist-packages/oaklib/utilities/lexical/lexical_indexer.py", line 316, in lexical_index_to_sssom
    Converter.from_prefix_map((meta or {}).pop(CURIE_MAP, {})),
AttributeError: 'tuple' object has no attribute 'pop'
```

**Comment by matentzn** [comment]
@twhetzel I think this is high priority as we cant run the mondo ingest pipeline at all now. My advice is to have Joe work on this, as he is familiar with both the mondo ingest and oaklib code.

**Comment by joeflack4** [comment]
Oh OK, looks like a simple bug to fix. I'll try my best to get in an OAK PR by Monday!

**Comment by joeflack4** [comment]
@matentzn Just wanting to know more information; how is this being run / what is the command / goal running this?

I think it is the goal: `$(MAPPINGSDIR)/mondo-sources-all-lexical.sssom.tsv`, and this is just one step that is erroring out as part of you just running the full pipeline, is that correct?

I worked on this a bit today. Most of my time went into fixing my OAK environment, and it looks like that's done.

First I tried to run this script in `mondo-ingest` and replicate before I went to fix it in OAK, but I got a different error, probably earlier in the script. This is all the time I have for this this weekend but I'll pick up from here next session:
`curies.api.DuplicateURIPrefixes:`
